be/src/exec/broker_writer.cpp

```diff
@@ -78,7 +78,8 @@ Status BrokerWriter::open() {
     TBrokerOpenWriterResponse response;
     try {
         Status status;
-        BrokerServiceConnection client(client_cache(_env), broker_addr, 10000, &status);
+        BrokerServiceConnection client(client_cache(_env), broker_addr, config::broker_write_timeout_seconds * 1000,
```

Comment:
Wha's the bug is? It's better to add a commit message about the problem and solution.

Reply:
add problem summary